### PR TITLE
1.20 Feat: Code Completion docs

### PR DESCRIPTION
**Pull request by Emyrk**
Feature in 1.20

### CODE REVIEW

File: workspaces/workspaces-as-code/intellisense.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Workspace template intellisense"
```
[the remark this answers is not in the document]
Comment (Emyrk) [reply workspaces/workspaces-as-code/intellisense.md:2]: I have no idea, didn't even think of that. I can change it

File: workspaces/workspaces-as-code/intellisense.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+description: "A better experience to creating workspace templates."
+state: beta
+---
+
+With 1.20 a feature was implemented that provides some level of intellisense when editing workspaces as code template files. This intellisense is provided through [json schemas](https://json-schema.org/).
+
+# Requirements
+
+- A coder deployment
```
[the remark this answers is not in the document]
Comment (Emyrk) [reply workspaces/workspaces-as-code/intellisense.md:11]: Sure

File: workspaces/workspaces-as-code/intellisense.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+At the top of your `coder.yaml` file add the following comment. Make sure to replace `<deployment_url>` with your Coder deployment url. Once you have this line in your file, you can begin to use the intellisense. 
+
+```yaml
+# yaml-language-server: $schema=https://<deployment_url>/api/private/template/schemas/wac.schema.json
+```
+
+
+Helpful Keyboard Shortcuts: (see extension page for more info)
+- Document Outlining (Ctrl + Shift + O)
```
[the remark this answers is not in the document]
Comment (Emyrk) [reply workspaces/workspaces-as-code/intellisense.md:25]: TIL. Probably?

**Comment by khorne3** [comment]
@Emyrk did you want to make any additional changes to this before I include w/ the release?

**Comment by Emyrk** [comment]
@khorne3 It's all good!

**Review by vapurrmaid — APPROVED** [review]

**Pull request merged**